resources/views/mail/attendance/report.blade.php

```diff
@@ -0,0 +1,5 @@
+@component('mail::message')
+
+RoboJackets' attendance report from {{ $export->start_time->format('l, n/j/Y \a\t g:iA') }} to {{ $export->end_time->format('l, n/j/Y \a\t g:iA') }} is available [here]({{ route('attendance.export', ['secret' => $export->secret]) }}). That link will only work once and will expire on {{ $export->expires->format('l, n/j/Y \a\t g:iA') }}. For help or to get a new link, please reply to this email.
```

Comment:
Assuming this is a timestamp.

```suggestion
RoboJackets' attendance report from {{ $export->start_time->format('l, n/j/Y \a\t g:iA') }} to {{ $export->end_time->format('l, n/j/Y \a\t g:iA') }} is available [here]({{ route('attendance.export', ['secret' => $export->secret]) }}). That link will only work once and will expire at {{ $export->expires->format('l, n/j/Y \a\t g:iA') }}. For help or to get a new link, please reply to this email.
```